src/Microsoft.AspNetCore.Rewrite/Internal/RedirectToWwwRule.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Rewrite.Logging;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNetCore.Rewrite.Internal
+{
+    public class RedirectToWwwRule : IRule
+    {
+        public int StatusCode { get; set; }
+        public bool AllowLocalhost { get; set; } = true;
```

Comment:
Bleh, I don't like how I designed this when I was an intern. Let's make StatusCode and AllowLocalhost private and set them in the constructor, like:
```csharp
public RedirectToWwwRule(int statusCode, bool allowLocalhost) ...
```

Reply:
That's I would have written it personally too. I was just trying to be consistent with RedirectToHttpsRule. Will change